src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -103,6 +107,11 @@ export class MdProgressSpinner implements OnDestroy {
     this._cleanupIndeterminateAnimation();
   }
 
+  /** Stroke width of the progress spinner. By default uses 10px as stroke width. */
+  @Input()
```

Comment:
This will still break if the consumer sets it to something like "abc". IMO the getter/setter here is overkill.